src/components/OptimizedImage.js

```diff
@@ -42,24 +43,52 @@ export default function OptimizedImage(props) {
     }
   }
 
-  return (
-    <>
-      <img
-        src={cdnUrl || src}
-        alt=".."
-        srcSet={srcSet}
-        sizes={sizes}
-        loading="lazy"
+  const fallbackMessage = (
+    <Alert
+      severity="error"
+      style={{
+        flexDirection: 'column',
+      }}
+    >
+      <div
         style={{
-          position: 'absolute',
-          inset: 0,
-          objectFit,
-          width: 'calc(100%)', // width for image rotation
-          height: 'calc(100%)', // height for image rotation
-          transform: `rotate(${rotation}deg)`,
+          display: 'flex',
+          flexDirection: 'column',
         }}
-        {...rest}
-      />
+      >
+        <AlertTitle>Failed to load image.</AlertTitle>
+        <p>
+          Please try — <strong>reloading the page.</strong>
+        </p>
+        <p>
+          Or — <strong>report the issue to an admin.</strong>
+        </p>
+      </div>
+    </Alert>
+  );
+
+  return (
+    <>
+      {!cdnUrl ? (
```

Comment:
This looks like it'll only display the error message when `cdnUrl` is undefined/null.
We also need to fall back when the url is defined but the image fails to load (e.g., error in url).

Reply:
Hey Nick, I added the native `onError` event handler for images. I set the event `target.src` to `null` in the handler logic, which should trigger the rendering of the fallback message. Tell me, what do you think about this?

Reply:
That works nicely.